### DISCUSSION TITLE
http://localhost:5173/%E7%AC%94%E8%AE%B0/%F0%9F%91%BE%2038965-machine-learning/svm.html

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/nolebase-template#32] [Announcements]
# http://localhost:5173/%E7%AC%94%E8%AE%B0/%F0%9F%91%BE%2038965-machine-learning/svm.html

记录回忆，知识和畅想的地方

http://localhost:5173/%E7%AC%94%E8%AE%B0/%F0%9F%91%BE%2038965-machine-learning/svm.html

**Comment by indexss** [comment]
一些行内公式未能渲染。
